src/pid.cpp

```diff
@@ -335,12 +336,13 @@ double Pid::computeCommand(double error, double error_dot, ros::Duration dt)
   // Calculate the integral of the position error
   i_error_ += dt.toSec() * p_error_;
 
-  if(gains.antiwindup_)
+  if(gains.antiwindup_ && gains.i_gain_!=0)
   {
     // Prevent i_error_ from climbing higher than permitted by i_max_/i_min_
-    i_error_ = boost::algorithm::clamp(i_error_,
-                                       gains.i_min_ / std::abs(gains.i_gain_),
-                                       gains.i_max_ / std::abs(gains.i_gain_));
+    double i_error_min = std::min<double>(gains.i_min_ / gains.i_gain_, gains.i_max_ / gains.i_gain_);
```

Comment:
Since we are using  `boost::algorithm`  for the clamping, we could use `<boost/algorithm/minmax.hpp>` as well.
I'd like to avoid this duplication..